packages/merch/lib/orderstatus.ts

```diff
@@ -0,0 +1,27 @@
+import { OrderStatus } from "types";
```

Comment:
not sure we should have a packages/merch dir. 
fyi this is causing build errors since theres no tsconfig in packages/merch.

2 courses of action:
* we remove this package
* we add flesh out this package with a tsconfig and package.json

im not quite sure if we need a merch package at this time.

@chanbakjsd could you chime in?